RecoTracker/TkHitPairs/interface/RecHitsSortedInPhi.h

```diff
@@ -39,15 +39,15 @@ class RecHitsSortedInPhi {
   typedef std::pair<HitIter,HitIter>            Range;
 
   using DoubleRange = std::array<int,4>;
-  
+
```

Comment:
All changes in this file are reformatting, so in principle could be avoided in this PR.